edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/authenticators/DeviceScopeTokenAuthenticator.cs

```diff
@@ -137,7 +137,7 @@ internal bool ValidateAudience(string audience, IIdentity identity)
             }
 
             if (string.IsNullOrWhiteSpace(hostName) ||
-                !(this.iothubHostName.Equals(hostName) || this.edgeHubHostName.Equals(hostName)))
+                !(string.Equals(this.iothubHostName, hostName, StringComparison.OrdinalIgnoreCase) || string.Equals(this.edgeHubHostName, hostName, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Is this correct? Are these case insensitive?

Reply:
is the same reasoning as when we made deviceId/moduleId check case-insensitive, SAS tokens come in as all lower case by convention, so we need to relax our string comparison logic on the hostname as well.